src/Sylius/Bundle/OrderBundle/Aggregator/AdjustmentsByLabelAggregator.php

```diff
@@ -23,12 +23,12 @@ class AdjustmentsByLabelAggregator implements AdjustmentsAggregatorInterface
      */
     public function aggregate(array $adjustments)
     {
-        $aggregatedAdjustments = array();
         foreach ($adjustments as $adjustment) {
-            if (!$adjustment instanceof AdjustmentInterface) {
-                throw new \InvalidArgumentException('Each adjustments array element must implement '.AdjustmentInterface::class.'.');
-            }
+            $this->assertElementIsAdjustment($adjustment);
```

Comment:
But this method doesn't modify the adjustments, just aggregates them, so even if it will aggregate 100 adjustments and 101th is incorrect then it will throw exception, so no harm. 

The case where the adjustment will not be of type `AdjustmentInterface` should only happen in DEV because of developer's mistake, so this unnecessary operations will not affect the performance of application, but iterating through the array 2 times each time the method is invoke will.